builder/bundles.go

```diff
@@ -360,7 +363,11 @@ func resolvePackages(numWorkers int, set bundleSet, packagerCmd []string, emptyD
 			// fail in the actual install to the full chroot.
 			outBuf, _ := helpers.RunCommandOutputEnv(queryString[0], queryString[1:], []string{"LC_ALL=en_US.UTF-8"})
 
-			rpm, fullRpm := repoPkgFromNoopInstall(outBuf.String())
+			rpm, fullRpm, e := repoPkgFromNoopInstall(outBuf.String())
+			if e != nil {
+				errorCh <- errors.Wrapf(e, bundle.Name)
```

Comment:
I would suggest also printing the error here for first hand info. Lets not wait for it to be trickled through the go channels. E.g. https://github.com/clearlinux/mixer-tools/blob/master/builder/update.go#L207